test/server/overload_manager_impl_test.cc

```diff
@@ -339,6 +411,46 @@ TEST_F(OverloadManagerImplTest, DuplicateOverloadAction) {
                           "Duplicate overload action .*");
 }
 
+TEST_F(OverloadManagerImplTest, RangeTriggerMaxLessThanMin) {
+  const std::string config = R"EOF(
+    resource_monitors {
+      name: "envoy.resource_monitors.fake_resource1"
+    }
+    actions {
+      name: "envoy.overload_actions.dummy_action"
+      triggers {
+        name: "envoy.resource_monitors.fake_resource1"
+        range {
+          min_value: 0.9
+          max_value: 0.8
+        }
+      }
+    }
+  )EOF";
+
+  EXPECT_THROW_WITH_REGEX(createOverloadManager(config), EnvoyException, "min_value.*max_value.*");
+}
+
+TEST_F(OverloadManagerImplTest, RangeTriggerMaxEqualsMin) {
+  const std::string config = R"EOF(
+    resource_monitors {
+      name: "envoy.resource_monitors.fake_resource1"
+    }
+    actions {
+      name: "envoy.overload_actions.dummy_action"
+      triggers {
+        name: "envoy.resource_monitors.fake_resource1"
+        range {
+          min_value: 0.9
+          max_value: 0.9
+        }
+      }
+    }
+  )EOF";
+
+  EXPECT_THROW_WITH_REGEX(createOverloadManager(config), EnvoyException, "min_value.*max_value.*");
```

Comment:
nit: You may as well write out "min_value must be less than max_value" so the person looking at this exception has an easier time looking for the corresponding throw.